doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -26,23 +26,23 @@ Optionally, you can set custom configuration for {KIE_SERVERS}. In this case, de
 +
 For instructions about creating custom images, see xref:customimage-con_{context}[].
 +
-. If you want to configure an immutable {KIE_SERVER}, complete the following additional steps:
+. If you want to configure an immutable {KIE_SERVER} using a Source to Image (S2I) build, complete the following additional steps:
++
+[IMPORTANT]
+====
+If you want to configure an immutable {KIE_SERVER} that pulls services from the Maven repository, do not click *Set Immutable server configuration* and do not complete these steps. Instead, set the `KIE_SERVER_CONTAINER_REPLOYMENT` environment variable.
```

Comment:
```suggestion
If you want to configure an immutable {KIE_SERVER} that pulls services from the Maven repository, do not check *Set Immutable server configuration* checkbox and do not complete these steps. Instead, set the `KIE_SERVER_CONTAINER_REPLOYMENT` environment variable.
```

Reply:
![image](https://user-images.githubusercontent.com/33361640/99993740-04071880-2db0-11eb-882d-e7d4c904dbb4.png)

It is a button not a check box though?

Reply:
ok, so, click on the "..." button and...?